pkg/restapi/rp/operation/operations.go

```diff
@@ -22,7 +22,14 @@ import (
 const (
 	verifyVC            = "/verifyVC"
 	httpContentTypeJSON = "application/json"
-	vcsVerifyEndpoint   = "/verify"
+
+	// edge-service endpoint to verify presentation
+	vcsVerifyEndpoint = "/verify"
+
+	// edge-service endpoint to verify presentation
+	vpVerifyEndpoint = "/verifyPresentation"
+
+	verifyVP = "/verifyVP"
```

Comment:
What is the difference between two?

Reply:
One was endpoint of edge-service and other was endpoint in edge sandbox ..thats how it was done for verify credentials. But i noticed its confusing there for i changed for both in edge-sandbox.